linode/manager
# refactor: [M3-7441] - Clean up NodeBalancer Firewall Feature Flag

## Description 📝

- Cleans up the NodeBalancer Firewall feature flag (`firewallNodebalancer`)  because this feature has been out for a while 🧹🧱🔥 

## Preview 📷

> [!note]
> No UI changes expected

## How to test 🧪

- Check out this PR locally or use the internal preview link 🔗 
- Check general functionality of NodeBalancers and Firewalls
- Verify code changes do not introduce any breaking changes 

## As an Author I have considered 🤔
